server/simplehttp/client.go

```diff
@@ -0,0 +1,34 @@
+package simplehttp
+
+import (
+	"net"
+	"net/http"
+	"time"
+)
+
+type SimpleClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+// NewClient creates a SimpleClient intended for one-off requests, like getPushProxyVersion.
+// If we end up needing something more long term, we should consider storing it.
```

Comment:
Can't we simply use `http.DefaultClient`?

Reply:
Unless things have changed, it used to be that `http.DefaultClient` shouldn't be used in production (it's missing request timeouts, at least)

Reply:
We could set `http.DefaultClient.Timeout` in a`init()` function. It's just that adding a whole new package and instantiate a new client for a single request seems a bit overkill.

Reply:
K, done, but I'm not totally sure about having it in main (because we're creating mocks right inj). It is simpler though, so good enough for me.

Reply:
I guess the question is, do we really need mocks? It's just an HTTP client, I am thinking using [`httptest`](https://pkg.go.dev/net/http/httptest) would have been sufficient?

Reply:
Yah, you're right -- fixed, thanks.